lab-elizabeth/app/component/gallery/edit-gallery/edit-gallery.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  template: require('./edit-gallery.html'),
+  controller: ['$log', 'galleryService', EditGalleryController],
+  controllerAs: 'editGalleryCtrl',
+  bindings: {
+    gallery: '<',
+  },
+};
+
+function EditGalleryController($log, galleryService){
+  $log.debug('init editGalleryCtrl');
+
+  this.updateGallery = function(){
+    galleryService.updateGallery(this.gallery._id, this.gallery);
```

Comment:
You could probably refactor this to just take this.gallery and access the _id property on that within galleryService.updateGallery, but this works just fine.